src/OpenTracing/ActiveSpanSource.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace OpenTracing;
+
+/**
+ * Keeps track of the current active `Span`.
+ */
+interface ActiveSpanSource
+{
+    const THREAD_DEFAULT = 'default';
+
+    /**
+     * Activates an `Span`, so that it is used as a parent when creating new spans.
+     * The implementation must keep track of the active spans sequence, so
+     * that previous spans can be resumed after a deactivation.
+     *
+     * @param Span $span
+     * @param string|array $threads Use only in case of async operations. Allows you to keep
+     * an span as active in different threads.
+     */
+    public function activate(Span $span, $threads = self::THREAD_DEFAULT);
```

Comment:
Putting the concepts of threads into this sounds weird given that PHP is completely single-threaded and there is no reliable cross-platform thread extension as far as I know.